packages/flutter_tools/test/commands.shard/hermetic/config_test.dart

```diff
@@ -285,37 +255,26 @@ void main() {
         '--analytics',
       ]);
 
-      expect(testUsage.enabled, true);
-
       // Verify that we only send the analytics enable event, and no other
       // info.
-      expect(testUsage.events, equals(<TestUsageEvent>[
-        const TestUsageEvent('analytics', 'enabled', label: 'true'),
-      ]));
-      expect(testUsage.commands, isEmpty);
-      expect(testUsage.timings, isEmpty);
       expect(fakeAnalytics.sentEvents, isEmpty);
     }, overrides: <Type, Generator>{
-      Usage: () => testUsage,
     });
 
     testUsingContext('analytics reported with help usages', () async {
       final ConfigCommand configCommand = ConfigCommand();
       createTestCommandRunner(configCommand);
 
-      testUsage.suppressAnalytics = true;
       expect(
         configCommand.usage,
         containsIgnoringWhitespace('Analytics reporting is currently disabled'),
       );
 
-      testUsage.suppressAnalytics = false;
       expect(
         configCommand.usage,
         containsIgnoringWhitespace('Analytics reporting is currently enabled'),
       );
     }, overrides: <Type, Generator>{
-      Usage: () => testUsage,
     });
```

Comment:
@christopherfujino/@jayoung-lee (Only if you know the answer off the top of your head) Do you happen to know what the intent here was? Was the idea here that 1) the first run of `flutter` always has telemtry disabled and 2) telemetry is enabled in the second run (unless user disables it first).

Reply:
Oh never mind, looking at the diff I see why this test is broken 😆 (I made this comment from VSCode, so I didnt' see the diff).